jax/_src/numpy/ufuncs.py

```diff
@@ -885,9 +981,52 @@ def tanh(x: ArrayLike, /) -> Array:
   """
   return lax.tanh(*promote_args_inexact('tanh', x))
 
-@implements(np.arctanh, module='numpy')
+
 @partial(jit, inline=True)
 def arctanh(x: ArrayLike, /) -> Array:
+  r"""Calculate element-wise inverse of hyperbolic tangent of input.
+
+  JAX implementation of :obj:`numpy.arctanh`.
+
+  The inverse of hyperbolic tangent is defined by:
+
+  .. math::
+
+    arctanh(x) = \frac{1}{2} [ln(1 + x) - ln(1 - x)]
+
+  Args:
+    x: input array or scalar.
+
+  Returns:
+    An array of same shape as ``x`` containing the inverse of hyperbolic tangent
+    of each element of ``x``, promoting to inexact dtype.
+
+  Note:
+    - ``jnp.arctanh`` returns ``nan`` for real-values outside the range ``[-1, 1]``.
+    - ``jnp.arctanh`` is equivalent to computing ``1j * jnp.arctan(x / 1j)``.
```

Comment:
I would remove this: we don't need to note well-known mathematical identities.

Reply:
Should I remove this in `arcsinh` doc also?

Reply:
Yes, I think so

Reply:
Thanks! Removed.